packages/cli-e2e/entrypoints/dockerX11Entry.sh

```diff
@@ -9,5 +9,10 @@ cd /home/cli-copy
 
 sudo -u notGroot npm run setup
 cd packages/cli-e2e
+
+mkdir dotenv
+echo "process.env.PLATFORM_USER_NAME='$PLATFORM_USER_NAME'" > dotenv/config.ts
```

Comment:
Similar to my question below, why do we need to set that in a dotenv file/config file ? 
Is that for jest ?

If they are set as env variable when we spawn docker, could we just pick them up inside jest ?

Reply:
Yes it's for Jest. 
And no, I was not able to use env variables in the tests without installing `dotenv`.

Instead, I used the `setupFiles` parameter to read the `dotenv/config.ts` file.